tests/test_app.py

```diff
@@ -0,0 +1,235 @@
+import datetime
+from unittest.mock import patch
+
+from freezegun import freeze_time
+from page_analyzer.utils.db import (
+    create_check,
+    create_url,
+    get_url_by_name,
+    get_url_checks,
+    get_url_data,
+)
+
+
+def test_index(client):
+    # given
+    route = "/"
+
+    # when
+    response = client.get(route)
+
+    # then
+    assert response.status_code == 200
+    assert "Бесплатно проверяйте сайты на SEO-пригодность" in response.text
```

Comment:
wdyt, is it enough to know that we return the stable and reproducible page content?